src/debug_menu.cpp

```diff
@@ -956,8 +956,24 @@ void character_edit_menu()
             }
         }
         break;
-        case D_FLU: {
-            p.add_effect( effect_flu, 1000_minutes );
+        case D_ADD_EFFECT: {
+            const auto text = string_input_popup()
+                              .title( _( "Choose an effect to add." ) )
+                              .width( 20 )
+                              .text( "" )
+                              .only_digits( false )
+                              .query_string();
+            efftype_id effect( text );
+            int intensity;
+            int seconds;
```

Comment:
```suggestion
            int seconds = 0;
```
Same for the other integer. If the queries are cancelled (`query_int` returns false), the values are not changed and remain unspecified. Reading them later results in undefined behaviour.